content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,296 @@ weight: 50
+    - address: 10.128.0.17
+      ports:
+        http: 8080
+      labels:
+        app: vm1
+        version: 1
+EOF
 
-Istio auth node agent is healthy:
-
-{{< text bash >}}
-$ sudo systemctl status istio-auth-node-agent
-● istio-auth-node-agent.service - istio-auth-node-agent: The Istio auth node agent
-   Loaded: loaded (/lib/systemd/system/istio-auth-node-agent.service; disabled; vendor preset: enabled)
-   Active: active (running) since Fri 2017-10-13 21:32:29 UTC; 9s ago
-     Docs: https://istio.io/
- Main PID: 6941 (node_agent)
-    Tasks: 5
-   Memory: 5.9M
-      CPU: 92ms
-   CGroup: /system.slice/istio-auth-node-agent.service
-           └─6941 /usr/local/istio/bin/node_agent --logtostderr
-
-Oct 13 21:32:29 demo-vm-1 systemd[1]: Started istio-auth-node-agent: The Istio auth node agent.
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.469314    6941 main.go:66] Starting Node Agent
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.469365    6941 nodeagent.go:96] Node Agent starts successfully.
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.483324    6941 nodeagent.go:112] Sending CSR (retrial #0) ...
-Oct 13 21:32:29 demo-vm-1 node_agent[6941]: I1013 21:32:29.862575    6941 nodeagent.go:128] CSR is approved successfully. Will renew cert in 29m59.137732603s
 {{< /text >}}
 
-## Running services on a mesh expansion machine
+## Troubleshooting
+
+The following are some basic trouble shooting steps for common mesh expansion issues.
+
+*    When making requests from a VM to the cluster, ensure you don't run the requests as `root` or
+    `istio-proxy` user. By default, Istio excludes both users from interception.
+
+*    Verify the machine can reach the IP of the all workloads running in the cluster. For example:
 
-*   Configure the sidecar to intercept the port. This is configured in `/var/lib/istio/envoy/sidecar.env`,
-using the `ISTIO_INBOUND_PORTS` environment variable.
+    {{< text bash >}}
+
+    $ kubectl get endpoints -n bookinfo productpage -o jsonpath='{.subsets[0].addresses[0].ip}'
+    10.52.39.13
 
-    Example (on the VM running the service):
+    {{< /text >}}
 
     {{< text bash >}}
-    $ echo "ISTIO_INBOUND_PORTS=3306,8080" > /var/lib/istio/envoy/sidecar.env
-    $ systemctl restart istio
+
+    $ curl 10.52.39.13:9080
+    html output
+
     {{< /text >}}
 
-*   Manually configure a selector-less service and endpoints. The `selector-less` service is used for
-services that are not backed by Kubernetes pods.
+*    Check the status of the node agent and sidecar:
 
-    Example, on a machine with permissions to modify Kubernetes services:
+    {{< text bash >}}
+
+    $ sudo systemctl status istio-auth-node-agent
+    $ sudo systemctl status istio
+
+    {{< /text >}}
+
+*    Check that the processes are running:
 
     {{< text bash >}}
-    $ istioctl -n onprem register mysql 1.2.3.4 3306
-    $ istioctl -n onprem register svc1 1.2.3.4 http:7000
+
+    $ ps aux |grep istio
```

Comment:
At first glance, readers don't know where the command and output start/end.
Consider splitting them into two code blocks and signal to the reader what they should look for in the output.

Reply:
Actually the Istio theme does a nice job of formatting bash code blocks so you can easily see where the command is - the output (any lines that don't begin with $) has a different background colour. I can change it but I'm not sure if it's necessary given the presentation.

Reply:
Correct, throughout the doc set, we encourage the command and its output to be in the same block, and we format it correctly in a consistent way.